package.json

```diff
@@ -338,6 +338,9 @@
         }
       }
     ],
+    "configurationDefaults": {
+      "editor.formatOnType": true
+    },
```

Comment:
What does this change do?

Reply:
@scottdover 
cc: @scnwwu
This is to enable executing our codes in the LSP server to insert indentations into the next line when a user types "\n". See: https://code.visualstudio.com/api/language-extensions/programmatic-language-features#incrementally-format-code-as-the-user-types

Reply:
@scnwwu I totally agree, it's reasonable.